Makefile.vofiles

```diff
@@ -28,6 +28,14 @@ VFILES := $(THEORIESV) $(CONTRIBV)
 # Build directory is needed to have the glob files
 BUILD_VFILES=$(addprefix $(VO_OUT_DIR),$(VFILES))
 
+BUILD_VFILES_FILE=$(BUILD_OUT_DIR).all_v_files
+
+# This has to be kept in sync with above, used in the windows build to
+# avoid coqdep maximum number arguments problems. Paths in file have
+# to be relative to the location of the coqdep input file.
+$(BUILD_VFILES_FILE): $(BUILD_VFILES)
+	$(HIDE)find theories user-contrib -type f -name "*.v" -fprintf "$@" "lib/coq/%p\n"
```

Comment:
osx doesn't have printf, that's why I sed

Reply:
Damn, while writing the patch I thought of checking that but I was lazy. But the situation seems a f***-up, all of `find` `sed` `xargs` and `stat` are different in OSX so I've opted for using a specific OSX command.